README.md

```diff
@@ -1,3 +1,9 @@
 # Using Endpoints on Google Compute Engine or Container Engine
```

Comment:
Are GKE/K8S called container engine? Sound not straightforward to me.

Reply:
Yes, GKE used to be "Google Container Engine", now it changed to "Google Kubernetes engine".  I will update that